ibm_catalog.json

```diff
@@ -208,21 +213,23 @@
           "dependencies": [
             {
               "name": "deploy-arch-ibm-account-infra-base",
+              "description": "This architecture supports creating and configuring the foundational components of an IBM Cloud account.",
```

Comment:
As discussed, please refer the latest updated ones and be consistent.
https://github.com/terraform-ibm-modules/terraform-ibm-kms-all-inclusive/blob/main/ibm_catalog.json#L275